source/WanManager/wanmgr_policy_autowan_impl.c

```diff
@@ -820,11 +871,18 @@ static ANSC_STATUS WanMgr_Trigger_Backup_TAD(DML_WAN_IFACE  *pFixedInterface, WC
     return retStatus;
 }
 
-static void WanMgr_Sysevent_SetConnectivityCheckState(int state)
+static void WanMgr_Sysevent_SetConnectivityCheckState(bool enable, int state)
```

Comment:
Please add a comment to explain the enable and state variables.

Reply:
ok